jxl/src/entropy_coding/context_map.rs

```diff
@@ -18,8 +18,7 @@ fn move_to_front(v: &mut [u8], index: u8) {
 }
 
 fn inverse_move_to_front(v: &mut [u8]) {
-    use array_init::array_init;
-    let mut mtf: [u8; 256] = array_init(|x| x as u8);
+    let mut mtf: [u8; 256] = std::array::from_fn(|x| x as u8);
```

Comment:
Can you also remove array_init from Cargo.toml?

Reply:
Not entirely because of `array_init::try_array_init`.

https://github.com/libjxl/jxl-rs/blob/36ba4e8ceef778e0b514f7439447cfac7312cd2c/jxl/src/headers/encodings.rs#L196

There is a standard library alternative but it is nightly-only -> https://github.com/rust-lang/rust/issues/89379